index.html

```diff
@@ -44,9 +53,9 @@ <h2>Connect:</h2>
         <li>https://www.instagram.com/gemcrate/</li>
       </ul>
     </section>
-    <section>
+    <section id="leave_message">
```

Comment:
^Yeah like this is perfect, you have the id in the section element, which would make sense to me